tools/expander.go

```diff
@@ -0,0 +1,207 @@
+package tools
+
+import (
+	"encoding/json"
+	"errors"
+	"reflect"
+	"strings"
+
+	"github.com/conductorone/conductorone-sdk-go/pkg/models/shared"
+)
+
+const (
+	atTypeApp             = "type.googleapis.com/c1.api.app.v1.App"
```

Comment:
do we export these const from the SDK anywhere?  shouldn't we?

Reply:
I exported them